code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -6,7 +6,29 @@
 		tally -= 2
 
 	if(species.slowdown)
-		tally = species.slowdown
+		if(src.species.name == "Wryn")		//handles Wryn specific movement
+			if(!istype(get_turf(src), /turf/space))		//Space is 20c whut..
+				var/loc_temp = T0C
+				var/datum/gas_mixture/environment = loc.return_air()
+				if(istype(src.loc, /obj/mecha))
+					var/obj/mecha/M = src.loc
+					loc_temp =  M.return_temperature()
+				else
+					loc_temp = environment.temperature
+
+				switch(loc_temp)			//messy switch statement to assign the proper speed

```

Comment:
Jesus christ, no no no

Meth, the superfast chem, at most gives you +2 to the tally

This is terribad